tests/setup/resize_observer.mock.ts

```diff
@@ -0,0 +1,35 @@
+class MockResizeObserver {
+  public cb: Function;
+  constructor(cb: Function) {
+    this.cb = cb;
+  }
+  observe() {
+    //@ts-ignore
+    global.resizers.add(this);
```

Comment:
isn't that a memory leak ?
The reference to the callback and hence to the component is kept alive here